handlers/common.go

```diff
@@ -785,3 +781,36 @@ func getExecutionChartData(indices []uint64, currency string, lowerBoundDay uint
 
 	return chartData, nil
 }
+
+func GetTotalWithdrawalsCount(validators []uint64) (uint64, error) {
```

Comment:
this method should be moved to the db package

Reply:
fixed